core/network/protocols/propagate_transactions_protocol.cpp

```diff
@@ -300,34 +310,39 @@ namespace kagome::network {
     });
   }
 
-  //  void PropagateTransactionsProtocol::writePropagatedExtrinsics(
-  //      std::shared_ptr<Stream> stream,
-  //      const PropagatedExtrinsics &message,
-  //      std::function<void(outcome::result<std::shared_ptr<Stream>>)> &&cb) {
-  //    auto read_writer = std::make_shared<ScaleMessageReadWriter>(stream);
-  //
-  //    read_writer->write(message,
-  //                       [stream, wp = weak_from_this(), cb = std::move(cb)](
-  //                           auto &&write_res) mutable {
-  //                         auto self = wp.lock();
-  //                         if (not self) {
-  //                           stream->reset();
-  //                           if (cb) cb(ProtocolError::GONE);
-  //                           return;
-  //                         }
-  //
-  //                         if (not write_res.has_value()) {
-  //                           SL_VERBOSE(
-  //                               self->log_,
-  //                               "Error while writing propagated extrinsics:
-  //                               {}", write_res.error().message());
-  //                           stream->reset();
-  //                           if (cb) cb(write_res.as_failure());
-  //                           return;
-  //                         }
-  //
-  //                         if (cb) cb(stream);
-  //                       });
-  //  }
+  void PropagateTransactionsProtocol::propagateTransactions(
```

Comment:
looks like not covered at all by tests and also the most difficult part of the PR

Reply:
Hm. It was copy-pasted from gossiper implementation.